### PR TITLE
DEVDOCS-2143: Product Widget Tutorial

### PR DESCRIPTION
**Pull request by ektravel**
# [DEVDOCS-2143](https://jira.bigcommerce.com/browse/DEVDOCS-2143)

## What changed?
* added Product Widget Tutorial

### CODE REVIEW

File: docs/bigcommerce-for-wordpress/getting-started/install.md [hunk 1]
```diff
@@ -4,30 +4,30 @@
```
Comment (aglensmith) [inline docs/bigcommerce-for-wordpress/getting-started/install.md:4]: Did you mean to edit this file as a part of this pr?

Comment (ektravel) [reply docs/bigcommerce-for-wordpress/getting-started/install.md:4]: No, that change was submitted on PR [668](https://github.com/bigcommerce/dev-docs/pull/668)

File: docs/bigcommerce-for-wordpress/getting-started/install.md [hunk 2]
```diff
@@ -4,30 +4,30 @@
...
+### On this page
+- [System requirements:](#system-requirements)
 - [Installation](#installation)
-- [Setting Your Channel Name](#setting-your-channel-name)
+- [Setting your channel name](#setting-your-channel-name)
 
 </div>
 
-## System Requirements:
+## System requirements:
```
Comment (uwikaiddi) [inline docs/bigcommerce-for-wordpress/getting-started/install.md:14]:
```suggestion
## System requirements

```

File: docs/bigcommerce-for-wordpress/getting-started/install.md [hunk 3]
```diff
@@ -4,30 +4,30 @@
 
 <div class="otp" id="no-index">
 
-### On This Page
-- [System Requirements:](#system-requirements)
+### On this page
+- [System requirements:](#system-requirements)
```
Comment (uwikaiddi) [inline docs/bigcommerce-for-wordpress/getting-started/install.md:8]:
```suggestion
- [System requirements](#system-requirements)

```

File: docs/stencil-docs/page-builder/product-widget-tutorial.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+        "site_id": null,
+        "channel_id": 1
+    },
+    "meta": {}
+}
+```
+
+## Widget preview
+
+> You do not need to send a `POST` request to [Create a Widget](https://developer.bigcommerce.com/api-reference/store-management/widgets/widget/createwidget) or [Create a Widget Placement](https://developer.bigcommerce.com/api-reference/store-management/widgets/placement/createplacement) endpoints to reflect the new widget in Page Builder.
```
Comment (uwikaiddi) [inline docs/stencil-docs/page-builder/product-widget-tutorial.md:124]: @ektravel I suggest putting this note after the following sentence.

**Review by uwikaiddi — CHANGES_REQUESTED** [review]
@ektravel, I've included my suggestions. Please resolve the merge conflict as well.

**Review by uwikaiddi — CHANGES_REQUESTED** [review]

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**